Composer/packages/client/src/components/Notifications/SurveyNotification.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { useEffect } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import { ClientStorage } from '../../utils/storage';
+import { surveyEligibilityState, dispatcherState, machineInfoState } from '../../recoilModel/atoms/appState';
+import { MachineInfo } from '../../recoilModel/types';
+import { SURVEY_URL_BASE } from '../../constants';
+import TelemetryClient from '../../telemetry/TelemetryClient';
+
+function buildUrl(info: MachineInfo) {
+  // User OS
+  // hashed machineId
+  // composer version
+  // maybe include subscription ID; wait for global sign-in feature
+  // session ID  (global telemetry GUID)
+  const version = process.env.COMPOSER_VERSION;
+
+  const parameters = {
+    Source: 'Composer',
```

Comment:
Why is source capsized?

Reply:
I don't know. That's how it was in the spec I was given.